HelpSource/Classes/Method.schelp

```diff
@@ -40,14 +43,23 @@ instanceMethods::
 method::ownerClass
 returns::
 The link::Classes/Class:: for which the method is part of the implementation.
+If this is a class method, then the owner class will be a 'Meta_Class'.
 
 method::name
 returns::
 A link::Classes/Symbol:: which is the name of the Method.
 
 method::primitiveName
 returns::
-A link::Classes/Symbol:: which contains the name of the primitive function that implements the Method, if there is one.
+A link::Classes/Symbol:: which contains the name of the primitive function that implements the Method,
+if there is one, otherwise returns link::Classes/Nil::.
+
+method::defaultArgs
+Returns the default arguments needed to call the function.
```

Comment:
Returns the default arguments supplied in place of those not specified in a method call.